packages/components/atoms/f-button/src/components/Action.vue

```diff
@@ -0,0 +1,5 @@
+<template>
+    <button>
```

Comment:
Probably missed the change somewhere but is it possible to set this to `button=type` by default if the button component is used? To avoid setting it at component level.

Reply:
👍  thanks